resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Puffs out {PRONOUN/m_c/poss} chest, feeling secure with the space {PRONOUN/m_c/subject} {VERB/m_c/take/takes} up",
+            "Doesn't falter when challenged on {PRONOUN/m_c/poss} abilities",
+            "Offers tips to every cat in the Clan"
+        ],
+        "main_trait_constraint": [
+            "confident"
+        ]
+    },
+    {
+        "id": "cunning_gen",
+        "thoughts": [
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} easily could've gotten {PRONOUN/m_c/self}",
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
+            "Knows exactly how to get {PRONOUN/m_c/self} of a mess {PRONOUN/m_c/subject} orchestrated",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
+            "Got a clanmate to do {PRONOUN/m_c/poss} work never intending to pay {PRONOUN/r_c/object} back for it"
+        ],
+        "main_trait_constraint": [
+            "cunning"
+        ]
+    },
+    {
+        "id": "daring_gen",
+        "thoughts": [
+            "Is racing through camp, accidentally knocking into Clanmates",
+            "Took a nap on a high tree branch",
+            "Took a nap on the edge of the high rock",
+            "Suffered a bellyache after being dared to swallow a beetle",
+            "Is twitching {PRONOUN/m_c/poss} tail in excitement about something",
+            "Wishes there were more opportunities to show {PRONOUN/m_c/poss} skills",
+            "Defended a Clanmate in an argument"
+        ],
+        "main_trait_constraint": [
+            "daring"
+        ],
+        "main_status_constraint": [
+            "kitten",
+            "apprentice",
+            "medicine cat apprentice",
+            "mediator apprentice",
+            "warrior",
+            "medicine cat",
+            "mediator",
+            "deputy",
+            "leader",
+            "elder"
+            ]
+    },
+    {
+        "id": "daring_not_kit",
+        "thoughts": [
+            "Is letting a wound go untreated in hopes it gives {PRONOUN/m_c/object} a scar",
+            "Is leaping from boulder to boulder out in the forest",
+            "Is climbing to the top of a very tall tree!",
+            "Taunted a dog stuck behind a twoleg's fence",
+            "Is walking along the edge of a very tall cliff",
+            "Recently avoided a monster on the Thunderpath by the skin of {PRONOUN/m_c/poss} teeth!"
+        ],
+        "main_trait_constraint": [
+            "daring"
+        ],
+        "main_status_constraint": [
+            "apprentice",
+            "medicine cat apprentice",
+            "mediator apprentice",
+            "warrior",
+            "medicine cat",
+            "mediator",
+            "deputy",
+            "leader",
+            "elder"
+            ]
+    },
+    {
+        "id": "gen_faithful",
```

Comment:
We don't really consider "faithful" to always mean "loyal to StarClan", rather it means a cat would be faithful to their friends a beliefs, whatever those are.  You could take the StarClan specific thoughts here and move them to a thought group that requires the Star skill, I think that would make more sense with how we characterize faithful cats.

Reply:
Will remove the starclan thoughts until I properly understand adding skill thoughts